common/views/components/Tasl/Tasl.tsx

```diff
@@ -204,13 +207,22 @@ const Tasl: FunctionComponent<Props> = ({
     setIsActive(!isActive);
   }
 
+  const id = title
+    ? dasherizeShorten(title)
+    : sourceName
+    ? dasherizeShorten(sourceName)
+    : copyrightHolder
+    ? dasherizeShorten(copyrightHolder)
+    : '';
+  const idWithSuffix = `${id}${idSuffix}`;
```

Comment:
should there be a dash in between?

Reply:
I didn't think it was really necessary since it's mostly just a garbled string that's only required for machines.